groups/live/data.tf

```diff
@@ -96,4 +96,8 @@ data "template_cloudinit_config" "bulk_gateway_userdata_config" {
     content_type = "text/x-shellscript"
     content      = data.template_file.bulk_gateway_userdata.rendered
   }
+}
+
+data "aws_sns_topic" "bulk_gateway_sns" {
+  name = "bulk-gateway-sftp-cloudwatch-20220325131737971800000002"
```

Comment:
Can you use wildcard here to remove that long number?
The number is auto-generated so if it ever changes this would fail.

Reply:
wildcard bulk-gateway-sftp-cloudwatch-* was used but it didn't work:

Error: no matching SNS Topic found

  on data.tf line 101, in data "aws_sns_topic" "bulk_gateway_sns":
 101: data "aws_sns_topic" "bulk_gateway_sns" {

It only works when the full name is used.